Common/src/main/resources/assets/biomeswevegone/lang/ru_ru.json

```
@@ -75,6 +84,9 @@
   "biome.biomeswevegone.white_mangrove_marshes": "Белые мангровые болота",
   "biome.biomeswevegone.windswept_desert": "Ветреная пустыня",
   "biome.biomeswevegone.zelkova_forest": "Дзельковый лес",
+  "biomeswevegone.commands.reload.misc.success": "Успешно перезагружены разные конфигурации",
+  "biomeswevegone.commands.reload.spawn.success": "Успешно перезагружена конфигурация спавна мобов",
```

В официальном переводе Minecraft нет транслитерации "спавн"
```suggestion
  "biomeswevegone.commands.reload.spawn.success": "Успешно перезагружена конфигурация появления мобов",
```